ui-participant/src/util/surveyJsUtils.tsx

```diff
@@ -81,16 +83,20 @@ export function useSurveyJSModel(
   resumeData: SurveyJsResumeData | null,
   onComplete: () => void,
   pager: PageNumberControl,
-  profile?: Profile,
   opts: UseSurveyJsModelOpts = {}
 ) {
   const {
-    extraCssClasses = {}
+    extraCssClasses = {},
+    extraVariables = {}
   } = opts
 
   const { portalEnv } = usePortalEnv()
   const { i18n } = useI18n()
 
+  const { profile } = useActiveUser()
+  const { user, enrollees } = useUser()
+  const proxyProfile = enrollees.find(enrollee => enrollee.participantUserId === user?.id && enrollee.profile)?.profile
+
```

Comment:
consequence of this: profile/proxyProfile are available on _all_ surveys by default, don't need to worry about passing it down